pkg/gds/mock.go

```diff
@@ -67,6 +69,17 @@ func NewMock(conf config.Config) (s *Service, err error) {
 		return nil, err
 	}
 	svc.admin = admin
+
+	if conf.Sectigo.Testing {
+		if err = mock.Start(); err != nil {
```

Comment:
Not for this PR but we may need to change this to `sectigo_mock` or something so that it's more understandable in the future.